regression/scripts/AVL/test_AVL.py

```diff
@@ -84,7 +84,7 @@ def main():
 
     # lift coefficient check
     lift_coefficient              = results.conditions.cruise.aerodynamics.lift_coefficient[0]
-    lift_coefficient_true         = 0.59495841
+    lift_coefficient_true         = 0.59324447
```

Comment:
does this script use the .avl file in the regression?  I think we should include it since we upload it.

Reply:
I changed the resolution. The AVL file is not created each time because it would require you to run AVL- which cannot be done on travis. I updated the results scripts which the regression is built upon

Reply:
Gotcha, i think.  I think @timdmacdo knows more about this part than me.